modulefiles/module_base.wcoss_dell_p3

```diff
@@ -23,10 +23,14 @@ module load pm5
 
 #module load HDF5-serial/1.10.1
 #module load NetCDF/4.5.0
-module use /usrx/local/nceplibs/dev/NCEPLIBS/modulefiles
+#module use /usrx/local/nceplibs/dev/NCEPLIBS/modulefiles
```

Comment:
If the commented out modules are needed, remove them.   git tracks revisions so there is a record of what was present.

Reply:
@junwang-noaa can you please help me identify what can be removed in this file and what may need to be updated? Thanks.

Reply:
I removed commented out lines in module file. Will commit.